bats/core/api/merchant.bats

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bats
+
+load "../../helpers/_common.bash"
+load "../../helpers/user.bash"
+
+setup_file() {
+  clear_cache
+
+  create_user 'merchant'
+  user_update_username 'merchant'
+}
+
+HYDRA_PUBLIC_API="http://localhost:4444"
+HYDRA_ADMIN_API="http://localhost:4445"
+
+
+@test "admin: can retrieve admin token" {
```

Comment:
We already have a test for this so it can probably just be part of the `setup_file` or abstracted as a function

Reply:
yup I just copy and paste it. which option do you think is better? between moving to `setup_file` but where the code would be a test elsewhere (would be more obvious if something break), or having both place where this code run as part of a setup (less code duplicate, not obvious this is a test)?

Reply:
I think we can maybe treat it similar to how we have `login_user` in `user.bash`. Would mean not having an explicit test and simply having a function that has assertions within it